client/my-sites/themes/themes-selection.jsx

```diff
@@ -191,69 +194,95 @@ function bindGetPremiumThemePrice( state, siteId ) {
 	return ( themeId ) => getPremiumThemePrice( state, themeId, siteId );
 }
 
+function filterOutUniversalThemes( themes = [] ) {
+	return themes.filter( ( theme ) => {
+		return theme?.template !== 'blockbase';
```

Comment:
also note blockbase does not use blockbase as a template, so the parent theme "blockbase" is not filtered out.